internal/backup/volumesnapshotcontent_action.go

```diff
@@ -91,23 +84,9 @@ func (p *VolumeSnapshotContentBackupItemAction) Execute(item runtime.Unstructure
 		return nil, nil, "", nil, errors.WithStack(err)
 	}
 
-	backupOnGoing := snapCont.GetLabels()[velerov1api.BackupNameLabel] == label.GetValidName(backup.Name)
 	operationID := ""
 	var itemToUpdate []velero.ResourceIdentifier
 
-	// Only return Async operation for VSC created for this backup.
-	if backupOnGoing {
-		// The operationID is of the form <volumesnapshotcontent-name>/<started-time>
-		operationID = snapCont.Name + "/" + time.Now().Format(time.RFC3339)
-		itemToUpdate = []velero.ResourceIdentifier{
-			{
-				GroupResource: kuberesource.VolumeSnapshotContents,
-				Name:          snapCont.Name,
-			},
-		}
-
-	}
-
```

Comment:
I think we don't need to return variables for `operationID` and `itemToUpdate` since no async operation will be triggered.

Reply:
Modified.